sensorpush_to_statusdb.py

```diff
@@ -354,8 +455,18 @@ def main(
     sensorpush_db = couch["sensorpush"]
 
     for sd in sensor_documents:
+        # Check if there already is a document for the sensor & date combination
+        view_call = sensorpush_db.view("entire_document/by_name_and_date")[
+            sd.sensor_name, sd.start_date_midnight
+        ]
+
         sd_dict = sd.format_for_statusdb()
 
+        if view_call.rows:
+            sd_dict = SensorDocument.merge_with(sd_dict, view_call.rows[0].value)
+            sd_dict["_id"] = view_call.rows[0].value["_id"]
+            sd_dict["_rev"] = view_call.rows[0].value["_rev"]
```

Comment:
Why not put these in the `merge_with` method? It gets all of `view_call.rows[0].value` anyway.

Reply:
Yes! I will update that.